examples/math/trigonometryExample/src/ofApp.cpp

```diff
@@ -35,27 +35,26 @@ void ofApp::setup(){
 
 //--------------------------------------------------------------
 void ofApp::update(){
-	cout << rotationSpeed << endl;
     
     if (!ofGetMousePressed()) {//press the mouse to stop the angle from incrementing.
 		angle+=rotationSpeed;//at each update the angle get's incremented 
 	}
-	if (angle>=TWO_PI) { //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
-		angle=0;
-	}
-
+	 //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
+	angle = ofWrap(angle, 0, TWO_PI);
```

Comment:
This is not an equivalent replacement.

Reply:
ofWrap Wrapping will wrap around zero, not clamp.

Reply:
```
/// \brief Find a value within a given range, wrapping the value if it overflows.
///
/// If a value is between from and to, return that value.
/// If a value is NOT within that range, wrap it.
///
/// Example:
/// ~~~~{.cpp}
///		ofWrap(5, 0, 10); // Returns 5.
///		ofWrap(15, 0, 10); // Also returns 5.
///		ofWrap(-5, 0, 10); // Also returns 5.
/// ~~~~
///
/// \param value The value to map.
/// \param from The lower bound of the range.
/// \returns to The upper bound of the range.
```

Reply:
and the original was warping not clamping but only in one direction